spec/models/subject_reduction_spec.rb

```diff
@@ -6,6 +6,18 @@
     expect(build(:subject_reduction)).to be_valid
   end
 
+  it '#prepare' do
```

Comment:
what's this test doing?  can you provide a clear description?

Reply:
apologies - i was the shortcut syntax for the describe block in the it block. I've updated to make it clearer and used the standard rspec describe naming for testing instance methods, https://rspec.rubystyle.guide/#describe-the-methods